### DISCUSSION TITLE
[Esame 14 Luglio 2023] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#366] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/46229351-9bbb-40be-ab7f-df11f2903c0b)

**Comment by Elia-Belli** [comment]
**(a)** Calcoliamo prima le permutazioni inverse:

$$\sigma_1^{-1}=\begin{pmatrix}1&2&3&4&5&6&7&8&9\\
3&9&8&5&7&2&4&1&6\end{pmatrix}\ ;\ \sigma_2^{-1}=\begin{pmatrix}1&2&3&4&5&6&7&8&9\\
7&9&6&5&1&2&4&3&8\end{pmatrix}\ ;\ \sigma_3^{-1}=\begin{pmatrix}1&2&3&4&5&6&7&8&9\\
4&8&6&9&7&1&2&5&3\end{pmatrix}$$

$$\sigma_1^{-1}\sigma_2^{-1}\sigma_3^{-1}=\begin{pmatrix}1&2&3&4&5&6&7&8&9\\
7&8&9&1&5&4&6&3&2\end{pmatrix}$$

**(b)** Decomponiamo le permutazioni in cicli:

$$\sigma_1=(1\ 8\ 3)(2\ 6\ 9)(4\ 7\ 5)\Rightarrow \sigma_1\ è\ pari$$

$$\sigma_2=(1\ 5\ 4\ 7)(2\ 6\ 3\ 8\ 9)\Rightarrow \sigma_2\ è\ dispari$$

$$\sigma_3=(1\ 6\ 3\ 9\ 4)(2\ 7\ 5\ 8)\Rightarrow \sigma_3\ è\ dispari$$

L'ordine di una permutazione è dato dall'mcm degli ordini dei cicli in cui si decompone, sappiamo che un k-ciclo ha ordine k:
$$o(\sigma_1)=mcm(3,3,3)=3$$

$$o(\sigma_2)=mcm(4,5)=20$$

$$o(\sigma_3)=mcm(5,4)=20$$

**(c)** Solamente $\sigma_2$ e $\sigma_3$ hanno stessa struttura ciclica, allora sono le uniche permutazioni coniugate; troviamo $\alpha$ tale che $\sigma_3=\alpha \sigma_2 \alpha^{-1}$ che in tal caso è unica:

$$\alpha=\begin{pmatrix}1&2&3&4&5&6&7&8&9\\
2&1&3&5&7&6&8&9&4\end{pmatrix}$$

Questo perchè abbiamo il vincolo che $\sigma_3=\alpha \sigma_2 \alpha^{-1}=(\alpha(1)\alpha(5)\alpha(4)\alpha(7))(\alpha(2)\alpha(6)\alpha(3)\alpha(8)\alpha(9))$

**Reply by interestingmaneuver** [reply]
Per (a) si ha anche che $\sigma_1^{-1} \sigma_2^{-1} \sigma_3^{-1} = (\sigma_3 \sigma_2 \sigma_1)^{-1}$
